src/services/pocket-relayer.ts

```diff
@@ -227,7 +231,7 @@ export class PocketRelayer {
     const fallbackAvailable = blockchainAltruist ? true : false
 
     try {
-      if (!this.alwaysRedirectToAltruists) {
+      if (!this.altruistOnlyChains.includes(blockchainID) && !this.alwaysRedirectToAltruists) {
```

Comment:
because this is a second path after the if, the conditions of the if are to not enter it so we are negating the situation, (as in, if this is value is there, then skip the if)

Reply:
I see, then at-least 1 test case to cover the new scenario, i.e. blockchain in the list of altruists-only chains.

Reply:
done